src/consensus/other.md

```diff
@@ -0,0 +1,15 @@
+# Other
+
+> [!WARNING]
+> TODO: not curated v
```

Comment:
What does "not curated" mean here? Is this an important concept you want to contribute as a blueprint or not?

Reply:
No, it means I still didn't have time to look at it 😄